### PR TITLE
CHANGELOG: update changelog for 1.9.1 release

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

**Comment by liangwen12year** [comment]
[citest]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+[1.9.1] - 2022-08-05
+--------------------
+
+### Bug Fixes
```
Comment (richm) [inline CHANGELOG.md:7]: Please add a New Features section with none like this:
```
### New Features

- none

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+- argument_validator: fix IPRouteUtils.get_route_tables_mapping() for whitespace
+  sequence
+
+### Other Changes
+
+- network_state: improve state comparison for achieving idempotency
```
Comment (richm) [inline CHANGELOG.md:14]: I think the network state change is a bug fix, and the argument validator is something for the Other category

Comment (liangwen12year) [reply CHANGELOG.md]: But according to what this issue states, https://github.com/linux-system-roles/network/issues/507, "argument_validator: fix IPRouteUtils.get_route_tables_mapping() for whitespace  sequence" is more like a bugfix.

Comment (richm) [reply CHANGELOG.md]: ok - so two bug fixes then, and `- none` for New Features and Other Changes

**Review by richm — APPROVED** [review]

**Comment by liangwen12year** [comment]
[citest]

**Comment by richm** [comment]
You don't need citest for this.  We do not want to do citest for changes that do not affect the code.  It is better, when creating PRs for CHANGELOG or other non-code related items (e.g. things under .github), to use `[citest skip]` in the PR subject/title line, to tell the ci system not to test this.

**Pull request merged**